Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.17
+
+RUN apt-get update && apt-get -y dist-upgrade
+RUN apt-get -y install build-essential libssl-dev libffi-dev libblas3 libc6 liblapack3 gcc python3-dev python3-pip cython3
```

Comment:
I don't think there is any need for python related libraries. @shubham-chhimpa

Reply:
yes removed them